docs/features/contributor-insights.md

```diff
@@ -0,0 +1,33 @@
+---
+id: contributor-insights
+title: "Organize and Monitor Your Contributors"
+sidebar_label: "Contributor Insights"
+keywords:
+  - "contributor insights"
+  - "contributors"
+  - "organize"
+  - "Monitor Contributors"
+  - "Organize Contributors"
+  - "Contributor Contributor Insights  Management"
+---
+
+The Contributor Insights  feature enables you to categorize, monitor, and analyze different groups of contributors within open source projects. With the Contributor Insights  feature, gain granular insights into each contributor's activity and contributions.
+
+Monitor individual commit histories, track who is actively engaged, identify alumni or new contributors, and even compare the performance of contributors against each other. This encompasses a wide range of metrics such as code commits, created and reviewed Pull Requests (PRs), as well as issue creation and commenting.
+
+![Contributor Insights -page](../../static/img/Contributor Insights -page.svg)
+
+## How to Create a Contributor Insight Page
+
+After logging in, navigate to your [Contributor Insights  Page](https://app.opensauced.pizza/hub/Contributor Insights /).
```

Comment:
nit: whitespace

```suggestion
After logging in, navigate to your [Contributor Insights Page](https://app.opensauced.pizza/hub/Contributor Insights /).
```

that link also looks broken to me - going there on beta doesn't work. Maybe we need a redirect to the users personal workspace and to create new insight pages.

Reply:
Already fixed!